### PR TITLE
"Named" Volumes and Volumes in Compose vs CLI

### LINKED ISSUE #19034: Improve docs for different "kind" of volumes, and differences between compose/docker
The current documentation and man pages are unclear about;

- what a "named" volume is
- differences between docker and docker-compose w.r.t. bind-mounted volumes (compose accepts paths relative to the docker-compose.yml, but docker only accepts absolute paths).
- man pages are not "self contained", I.e., in many cases, the full documentation is only available on docs.docker.com

As reported on Twitter; https://twitter.com/mt_inside/status/683403251232944128

### PR DESCRIPTION
**Pull request by aditirajagopal**
Resolves #19034

Signed-off-by: Aditi Rajagopal arajagopal@us.ibm.com

### CODE REVIEW

**Comment by aditirajagopal** [comment]
ping @thaJeztah @moxiegirl - let me know if anything needs to be changed here, and if you want me to tackle anything else

**Comment by thaJeztah** [comment]
Thanks @aditirajagopal, I'm not sure these changes address https://github.com/docker/docker/issues/19034. I think we need to add more documentation around this, describing what a "named" volume is, and possibly add some documentation / better clarify in docker-compose that there's a difference. 

I don't think that "When using the Docker CLI" will make people understand that we mean "contrary to Docker Compose"

ping @moxiegirl do you have some pointers perhaps?

File: docs/userguide/dockervolumes.md [hunk 1]
```diff
@@ -39,18 +39,21 @@ containers that bypasses the [*Union File System*](../reference/glossary.md#unio
...
-application container.
+`docker create` and `docker run` command. You can create multiple data volumes
+by providing a name passed to the "-v" flag. Let’s mount a single volume now in
+our web application container.
+
 
     $ docker run -d -P --name web -v /webapp training/webapp python app.py
 
-This will create a new volume inside a container at `/webapp`.
+This will create a new volume, web, inside a container at `/webapp`, based off
```
Comment (MHBauer) [inline docs/userguide/dockervolumes.md]: The container is named web, not the volume.

**Comment by MHBauer** [comment]
[?new issue?] Is there docs for the `docker volume` command? I feel like that should be covered later in the docs for this page as well. Instead of creating a `volume container` now, shouldn't volumes be created directly?

**Comment by MHBauer** [comment]
I am not familiar with compose, is there an issue or PR over there? If not I'll try to read the docs and post a link in the originating issue here.

**Comment by thaJeztah** [comment]
I'm going to close this PR, because I don't think it fixes the issue. Thanks though for contributing @aditirajagopal; I just think this needs another approach :heart:

**Comment by aditirajagopal** [comment]
@thaJeztah @moxiegirl -- if this isn't good enough, can you please provide more constructive feedback on how this can be improved?

**Pull request closed without merging**